wisp/wisp.php

```diff
@@ -597,3 +680,155 @@ function wisp_ClientArea(array $params) {
         // Ignore
     }
 }
+
+/* Utility Functions */
+function getNodes(array $params, int $loc_id) {
+    /*
+        Fetches and returns all nodes with a specific location_id
+    */
+    $filteredNodes = Array();
+    $nodes = wisp_API($params, 'nodes/');
```

Comment:
The `nodes/` endpoint should be changed to `nodes` since wispv2 returns an error when using `nodes/`